lib/main_common.pm

```diff
@@ -2405,6 +2405,24 @@ sub load_security_tests_selinux {
     loadtest "security/selinux/chcat";
 }
 
+sub load_security_tests_cc {
+    # Setup environment for cc testing: 'audit-test' test suite setup
+    # Such as: download code branch; install needed packages
+    loadtest 'security/cc/cc_audit_test_setup';
+
+    # Run test cases of 'audit-test' test suite which do NOT need SELinux env
+    loadtest 'security/cc/audit_tools';
+
+    # Setup environment for cc testing: SELinux setup
+    # Such as: set up SELinux with permissive mode and specific policy type
+    loadtest 'security/selinux/selinux_setup';
+    loadtest 'security/cc/cc_selinux_setup';
+
+    # Run test cases of 'audit-test' test suite which do need SELinux env
+    # TODO: add more test cases: poo#93441
```

Comment:
I meant to delete the TODO-entry from source code completely. See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details

Reply:
Done.